### PR TITLE
ADDED Command class implemented and tested. The methods in this class…

### PR DESCRIPTION
**Pull request by earandap**
… must be redefined.

to run the test of this class must install Test::Exception  cpan lib.

### CODE REVIEW

File: lib/Command.pm [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+sub new {
+    my $class = shift;
+    bless {}, $class;
+}
+
+sub execute {
+     die "This method must be overridden by a subclass of __PACKAGE__";
+ }
+
+sub validate {
```
Comment (manuellerena) [inline lib/Command.pm]: -1 I think that we don't have to validate the command params.
The only validation that must be performed is the command first letter.

Comment (earandap) [inline lib/Command.pm]: FIXED

**Comment by manuellerena** [comment]
+1

**Comment by pauriarte** [comment]
define my $self and return in new();

**Comment by pauriarte** [comment]
1+

**Pull request merged**